src/Components/Books.js

```diff
@@ -40,28 +44,73 @@ export default function Books() {
   return (
     <>
       <div className="container">
-        <div className="my-4 border border-3 p-0">
```

Comment:
- [ ] [OPTIONAL ]Great styling however you shouldn't leave the gray space in the corners and top and bottom please make sure to remove it so it can be like any normal web app 
![Screenshot 2023-01-25 at 17 21 50](https://user-images.githubusercontent.com/61790587/214619051-c940ffec-9b3b-4e5f-a63d-318f11069cbc.png)